### ISSUE TITLE
Could i set password on backup file?

### ISSUE THREAD

**Issue by thearabbit** [schickling/laravel-backup#36]
Could i set password on backup file?

**Comment by schickling** [comment]
No, this feature isn't something on the roadmap. But actually sounds pretty cool. Feel free to work on this feature together with compression support as a pull request :)
